plaidml/bridge/openvino/tests/functional/shared_tests_instances/single_layer_tests/activation.cpp

```diff
@@ -56,6 +38,24 @@ const std::map<ActivationTypes, std::vector<std::vector<float>>> activationTypes
     {HSwish, {}},
     {SoftPlus, {}},
     {HSigmoid, {}},
+};
+
+const std::vector<InferenceEngine::Precision> f32Precision = {
+    InferenceEngine::Precision::FP32,
+};
+
+const std::map<ActivationTypes, std::vector<std::vector<float>>> activationF32Types = {
+    {Acos, {}},
+    {Acosh, {}},
+    {Asin, {}},
+    {Asinh, {}},
+    {Atan, {}},
+    {Atanh, {}},
+    {Cosh, {}},
+    {Floor, {}},
+    {Sinh, {}},
+    {Tan, {}},
+    {Erf, {}},
```

Comment:
Is there a plan to add fp16 and fp64 support for these ops?

Reply:
What needs to be done is IMO reasonably clear: Add a pass for lowering/legalizing code of the form
```
%out = "stdx.acosh"(%in) : (f16) -> f16
```
for these eltwise ops.

This would presumably be different for different target devices. E.g., for the CPU case being tested here, I suspect the correct approach is to convert this to fp32 (since there won't be fp16 intrinsics). But other devices may want to retain this as fp16 code. Since this test is running on CPU, this list is the ops that don't lower on CPU specifically. I don't know whether this list has any relationship whatsoever with which ops work on fp16 on any other device.

That said, I don't believe anyone is actually assigned to do this work yet.

Reply:
I think we might need to have different passes for different target devices to deal with fp16. We can start looking at the fp16 issues on CPU and then other target devices.